code/datums/quirks/neutral_quirks.dm

```diff
@@ -48,7 +48,7 @@
 	var/mob/living/carbon/human/human_holder = quirk_holder
 	human_holder.add_blocked_language(/datum/language/common)
 	if(ishumanbasic(human_holder))
-		human_holder.grant_language(/datum/language/uncommon)
+		human_holder.grant_language(/datum/language/uncommon, TRUE, TRUE, LANGUAGE_QUIRK)
```

Comment:
Named parameters for the TRUEs (which will make you have to named parameter LANGUAGE_QUIRK as well)